ContactCenterInsights/tests/Unit/V1/ContactCenterInsightsClientTest.php

```diff
@@ -83,6 +85,135 @@ private function createClient(array $options = [])
     }
 
+    /** @test */
+    public function bulkAnalyzeConversationsTest()
+    {
+        $operationsTransport = $this->createTransport();
+        $operationsClient = new OperationsClient([
+            'apiEndpoint' => '',
+            'transport' => $operationsTransport,
+            'credentials' => $this->createCredentials(),
+        ]);
+        $transport = $this->createTransport();
+        $gapicClient = $this->createClient([
+            'transport' => $transport,
+            'operationsClient' => $operationsClient,
+        ]);
+        $this->assertTrue($transport->isExhausted());
+        $this->assertTrue($operationsTransport->isExhausted());
+        // Mock response
+        $incompleteOperation = new Operation();
+        $incompleteOperation->setName('operations/bulkAnalyzeConversationsTest');
+        $incompleteOperation->setDone(false);
+        $transport->addResponse($incompleteOperation);
+        $successfulAnalysisCount = 1153322545;
+        $failedAnalysisCount = 1044285998;
+        $expectedResponse = new BulkAnalyzeConversationsResponse();
+        $expectedResponse->setSuccessfulAnalysisCount($successfulAnalysisCount);
+        $expectedResponse->setFailedAnalysisCount($failedAnalysisCount);
+        $anyResponse = new Any();
+        $anyResponse->setValue($expectedResponse->serializeToString());
+        $completeOperation = new Operation();
+        $completeOperation->setName('operations/bulkAnalyzeConversationsTest');
+        $completeOperation->setDone(true);
+        $completeOperation->setResponse($anyResponse);
+        $operationsTransport->addResponse($completeOperation);
+        // Mock request
+        $formattedParent = $gapicClient->locationName('[PROJECT]', '[LOCATION]');
+        $filter = 'filter-1274492040';
+        $analysisPercentage = 1.60975766E8;
+        $response = $gapicClient->bulkAnalyzeConversations($formattedParent, $filter, $analysisPercentage);
+        $this->assertFalse($response->isDone());
+        $this->assertNull($response->getResult());
+        $apiRequests = $transport->popReceivedCalls();
+        $this->assertSame(1, count($apiRequests));
+        $operationsRequestsEmpty = $operationsTransport->popReceivedCalls();
+        $this->assertSame(0, count($operationsRequestsEmpty));
+        $actualApiFuncCall = $apiRequests[0]->getFuncCall();
+        $actualApiRequestObject = $apiRequests[0]->getRequestObject();
+        $this->assertSame('/google.cloud.contactcenterinsights.v1.ContactCenterInsights/BulkAnalyzeConversations', $actualApiFuncCall);
+        $actualValue = $actualApiRequestObject->getParent();
+        $this->assertProtobufEquals($formattedParent, $actualValue);
+        $actualValue = $actualApiRequestObject->getFilter();
+        $this->assertProtobufEquals($filter, $actualValue);
+        $actualValue = $actualApiRequestObject->getAnalysisPercentage();
+        $this->assertProtobufEquals($analysisPercentage, $actualValue);
```

Comment:
Getting following the error here:

1) Google\Cloud\ContactCenterInsights\Tests\Unit\V1\ContactCenterInsightsClientTest::bulkAnalyzeConversationsTest
Failed asserting that 160975760.0 matches expected 160975766.0.

Reply:
This should be fixed in https://github.com/googleapis/gapic-generator-php/pull/585